src/pymovements/gaze/gaze_dataframe.py

```diff
@@ -266,6 +266,30 @@ def transform(
                 _check_n_components(self.n_components)
                 kwargs['n_components'] = self.n_components
 
+            if transform_method.__name__ in {'pos2vel', 'pos2acc'}:
+                if 'position' not in self.frame.columns and 'position_column' not in kwargs:
+                    if 'pixel' in self.frame.columns:
+                        raise pl.exceptions.ColumnNotFoundError(
+                            "Neither 'position' is in the columns of the dataframe: "
+                            f'{self.frame.columns} nor is the position column specified. '
```

Comment:
can you put the list of available columns at the end of the sentence after `[...] specified. `?
Something like: `Available dataframe columns are {self.frame.columns}`

Reply:
done